server/dive_tasks/tasks.py

```diff
@@ -346,6 +347,9 @@ def train_pipeline(
         # Completely separate directory from `root_data_dir`
         with tempfile.TemporaryDirectory() as _training_output_path:
             training_output_path = Path(_training_output_path)
+            training_results_path = training_output_path / "category_models"
+            training_results_path.mkdir()
```

Comment:
Hack to fix problem in viame/viame where some training scripts create this directory and others dont